templates/fundamental.eld

```diff
@@ -1,3 +1,6 @@
 fundamental-mode ;; Available everywhere
 
 (today (format-time-string "%Y-%m-%d"))
+(now (format-time-string "<%Y-%0m-%0d %a %H:%0M>"))
```

Comment:
I regularly use variable `now` in my projects. As well as many other developers too. This change will bring false positive suggestions and will be very annoying for every day usage. 
```suggestion
(NOW (format-time-string "<%Y-%0m-%0d %a %H:%0M>"))
```